protocol/storage/miner.go

```diff
@@ -493,16 +493,19 @@ type dealsAwaitingSealStruct struct {
 	SuccessfulSectors map[uint64]*sectorbuilder.SealedSectorMetadata
 	// Maps from sector id to seal failure error string.
 	FailedSectors map[uint64]string
+	// Maps from sector id to a sectors commitSector message
+	CommitmentMessages map[uint64]cid.Cid
 
 	onSuccess func(dealCid cid.Cid, sector *sectorbuilder.SealedSectorMetadata)
 	onFail    func(dealCid cid.Cid, message string)
 }
 
 func (sm *Miner) loadDealsAwaitingSeal() error {
 	sm.dealsAwaitingSeal = &dealsAwaitingSealStruct{
-		SectorsToDeals:    make(map[uint64][]cid.Cid),
-		SuccessfulSectors: make(map[uint64]*sectorbuilder.SealedSectorMetadata),
-		FailedSectors:     make(map[uint64]string),
+		SectorsToDeals:     make(map[uint64][]cid.Cid),
```

Comment:
QUESTION: Under what circumstances [do we need to stringify uint64](https://github.com/filecoin-project/go-filecoin/blob/master/actor/builtin/miner/miner.go#L510) before using as a map key?

Reply:
Not here. This only gets serialized to JSON as far as I can tell.

Reply:
Thanks.